data/events.json

```diff
@@ -0,0 +1,558 @@
+  },
+  {
+    "eventDbId": "6000000053",
+    "studyDbId": "6000000010",
+    "date": {
+      "discreteDates": ["2017-05-12"]
+    },
+    "eventDescription": "Coragen (active ingredient Chlortraniliprole) injected into subsurface drip system in the afternoon, approximately 2:00 PM.  Applied at rate 4.5 oz/ac, for total of approximately 6 oz.",
+    "eventParameters": [
+        {
+          "code": "CH_NAME",
+          "name": "chem_applic_name",
+          "unit": "text",
+          "value": "Coragen"
+        },
+        {
+          "code": "CHCD",
+          "name": "chem_applic_material",
+          "unit": "text",
+          "value": "Chlortaniliprole"
+        },
+        {
+          "code": "CH",
```

Comment:
I can't find this code in the ICASA data dictionary. There's a *Chemicals application amount* (CHAMT)?

Reply:
In the Management_info tab? 

![image](https://user-images.githubusercontent.com/464871/171966515-204854a8-17fe-415d-b4c5-f51b9cdfddc5.png)

Reply:
The code used here should actually be CHAMT. The CH code is an outdated linkage variable that shouldn't be used anymore (and hence doesn't show up in the Vmapper variable search).